Step 12.1 ADD:  Show the chapter rating

YagoAccenture opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

## Release Notes

- **New Features**
	- Integrated TMDB API functionality to retrieve and display episode ratings and images.
	- Added new UI components for episode rating and image in the details screen layout.

- **Bug Fixes**
	- Enhanced error handling for fetching TMDB episode details.

- **Documentation**
	- Updated method signatures and added comments for clarity on new functionalities.

- **Refactor**
	- Renamed method for consistency in the repository class.

These updates enhance the user experience by providing richer episode information and improving error management.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Commit:
Step 12.1 ADD:  Show the chapter rating

Commit:
Step 12.1 EDIT:
val tmdbRepository = TmdbReposirory(tmdbRemoteDataSource) -> TmdbRepository(tmdbRemoteDataSource)

PabloGrelaM reviewed (changes requested):
GetEpisodeDetailsUseCase and GetEpisodeTMDBDetailsUseCase should be the same

Commit:
Step 12.1 EDIT:
Unification and renaming of the usecase, aesthetic modification of DetailsActivity, to display the new data correctly

Commit:
Step 12.1 EDIT:
Separate use cases to leave the code as before

Commit:
Step 12.1 ADD:
Use cases

Commit:
Step 12.1 EDIT:
Testcases unified: GetEpisodeTMDBUseCase and GetEpisodeUseCase

Commit:
Step 12.1 EDIT:
Rename of use case UnificatedUseCase -> GetEpisodeAndDetailsUseCase

Commit:
Step 12.1 ADD:
Creation of an entity at the domain layer
